frontend/src/scenes/experiments/ExperimentView/CumulativeExposuresChart.tsx

```diff
@@ -0,0 +1,104 @@
+
+const getCumulativeExposuresQuery = (experiment: Experiment): InsightVizNode<TrendsQuery> => {
+    const experimentInsightType = experiment.filters?.insight || InsightType.TRENDS
+
+    // Trends Experiment
+    if (experimentInsightType === InsightType.TRENDS) {
+        const exposureName = experiment.parameters?.custom_exposure_filter?.events?.[0]?.name
+        const eventName = exposureName ?? '$feature_flag_called'
+        return {
+            kind: NodeKind.InsightVizNode,
+            source: {
+                kind: NodeKind.TrendsQuery,
+                dateRange: {
+                    date_from: experiment.start_date,
+                    date_to: experiment.end_date,
+                },
+                interval: 'day',
+                trendsFilter: {
+                    display: ChartDisplayType.ActionsLineGraphCumulative,
+                    showLegend: true,
+                    smoothingIntervals: 1,
+                },
+                series: [
+                    {
+                        kind: NodeKind.EventsNode,
+                        event: eventName,
+                        math: BaseMathType.UniqueUsers,
+                    },
+                ],
+                breakdownFilter: !exposureName
+                    ? {
+                          breakdown: `$feature/${experiment.feature_flag_key}`,
+                          breakdown_type: 'event',
+                      }
+                    : undefined,
+            },
+        }
+    }
+    // Funnel Experiment
+    return {
+        kind: NodeKind.InsightVizNode,
+        source: {
+            kind: NodeKind.TrendsQuery,
+            dateRange: {
+                date_from: experiment.start_date,
+                date_to: experiment.end_date,
+            },
+            interval: 'day',
+            trendsFilter: {
+                display: ChartDisplayType.ActionsLineGraphCumulative,
+                showLegend: true,
+                smoothingIntervals: 1,
+            },
+            series: [
+                {
+                    kind: NodeKind.EventsNode,
+                    event: experiment.filters?.events?.[0]?.name,
+                    math: BaseMathType.UniqueUsers,
+                },
+            ],
+            breakdownFilter: {
+                breakdown: `$feature/${experiment.feature_flag_key}`,
+                breakdown_type: 'event',
+            },
+        },
+    }
+}
+
+export function CumulativeExposuresChart(): JSX.Element {
+    const { experiment } = useValues(experimentLogic)
+
+    return (
+        <div>
+            <div>
+                <h2 className="font-semibold text-lg">Cumulative exposures</h2>
+            </div>
+            {experiment.start_date ? (
+                <Query
+                    query={getCumulativeExposuresQuery(experiment)}
+                    readOnly
+                    context={{
+                        emptyStateHeading: 'No exposures to cumulate yet',
```

Comment:
I'd suggest a simpler "No exposures to show yet"

Reply:
cd0fbd0b13eea1b4cc61b3c19f9b90ddf38fd64d